### ISSUE TITLE
Creating opentelekomcloud_rds_instance_v3 with OTC provider v1.21.2 crashes TF, but works with v1.21.1

### ISSUE THREAD

**Issue by almo-ltoeroek** [opentelekomcloud/terraform-provider-opentelekomcloud#695]
# Terraform Version
Terraform v0.13.4
+ provider registry.terraform.io/hashicorp/helm v1.3.2
+ provider registry.terraform.io/hashicorp/kubernetes v1.12.0
+ provider registry.terraform.io/hashicorp/random v2.3.1
+ provider registry.terraform.io/opentelekomcloud/opentelekomcloud v1.21.2

# Affected Resource(s)
- opentelekomcloud_rds_instance_v3

# Terraform Configuration Files
```hcl
resource "opentelekomcloud_rds_instance_v3" "postgres" {
  name              = "my-postgres"
  availability_zone = ["eu-de-01"]
  flavor            = "rds.pg.c2.medium"

  vpc_id    = opentelekomcloud_vpc_v1.altermobili.id
  subnet_id = opentelekomcloud_vpc_subnet_v1.private.id

  security_group_id = opentelekomcloud_networking_secgroup_v2.postgres_secgroup.id

  db {
    password = random_password.postgres_root_password.result
    type     = "PostgreSQL"
    version  = "11"
  }
  volume {
    type = "COMMON"
    size = 40
  }
}

```

# Debug Output/Panic Output
https://gist.github.com/almo-ltoeroek/b439ab43a61ce55ac41725f8a2e36811

# Steps to Reproduce
1. `terraform apply`

# Expected Behavior
Create RDS resource

# Actual Behavior
TF CRASHes

# Important Factoids
Work with v1.21.1 but not with v1.21.2, I suspect the issue is related to https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/commit/09ff99ab0d367d07991b0a3a37bcdcb58b27630e

**Comment by lego963** [comment]
@almo-ltoeroek this problem cause you don't have `backup_strategy` and `port` this will be fixed in next release. Could you please for temporary add `backup_strategy` and `port`?

**Comment by outcatcher** [comment]
Fixed in #702 
Will be part of `v1.21.4`
